hedera-node/docs/maintainers-guide.md

```diff
@@ -1,14 +1,18 @@
 # Maintainers Guide
 This document outlines the process and keynotes for the core maintainers of this repository .
 
+## JDK
+Download JDK-17.0.3+ for mac [here](https://adoptium.net/temurin/releases/?os=mac&version=17).
```

Comment:
Java 21 is not currently supported but will be the minimum version required in mid-December.

Reply:
I'll try to remember to fix this up next month.